how-to/how-to-add-telerik-ui-for-nativescript-package.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to Include the Telerik UI for NativeScript Package
+description: Tutorial on how to include the Telerik UI for NativeScript package to AppBuilder project
+type: how-to
+page_title: How to add the Telerik UI for NativeScript package to an AppBuilder project
+slug: include-nativescript-ui-package
+position:
+tags: appbuilder, nativescript
+teampulseid:
+ticketid:
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package management</td>
+  </tr>
+  <tr>
+    <td>Mobile Development type</td>
+    <td>Native (NativeScript)</td>
+  </tr>
+</table>
+
+## Description
+This article provides the general steps required to include the Telerik UI for NativeScript package in your project.
+
+## Solution
+#### Telerik UI for NativeScript (Free)
+1. Open your project in AppBuilder.
+1. In the **Project Navigator** (**Solution Explorer** in Visual Studio), right-click the second node from the top and select **Manage Packages**.
+1. Find the `nativescript-telerik-ui` package inside the **NPM** panel and click **Install**.
+
+#### Telerik UI for NativeScript (Pro)
+1.	Navigate to `www.telerik.com/account` and log in your account.
+1.  Under **Products and Subscriptions**, select **UI for NativeScript**.
+1.  In the newly opened window click on **Download**.
+1.  Select the `Telerik_UI_for_NativeScript.zip` file under **Other Setup Files**.
+1.	Extract `nativescript-ui-pro.tgz` from the **Packages** folder of the downloaded file.
+1.	Open an AppBuilder client and load your project.
```

Comment:
Run AppBuilder and open your app.

Mostly, we're using app instead of project. Probably the sole exception are VS projects.